jsapi/src/main.js

```diff
@@ -721,6 +766,8 @@ lux.Map.prototype.addFeature = function(json) {
     return;
   }
   var size = this.getSize();
+  // remove any highlighted feature
+  this.showLayer_.getSource().clear();
```

Comment:
`addFeature` may be called several times (cf [foreach](https://github.com/Geoportail-Luxembourg/geoportailv3/pull/1481/files#diff-80f6c764111465dfaa94038b75e98be0L695)),  `clear` should maybe better done in `showFeatures`…